auth-service/src/main/java/com/petclinic/authservice/businesslayer/UserServiceImpl.java

```diff
@@ -103,11 +107,13 @@ public User createUser(@Valid UserIDLessRoleLessDTO userIDLessDTO) {
             user.setPassword(passwordEncoder.encode(user.getPassword()));
 
             log.info("Sending email to {}...", userIDLessDTO.getEmail());
-            log.info(mailService.sendMail(generateVerificationMail(user)));
+           // log.info(mailService.sendMail(generateVerificationMail(user)));
             log.info("Email sent to {}", userIDLessDTO.getEmail());
 
-            return userRepo.save(user);
+            User savedUser = userRepo.save(user);
+            CartResponse cartResponse = cartService.createCart(new CartRequest(savedUser.getUserIdentifier().getUserId()));
 
+            return savedUser;
```

Comment:
I'm interested in knowing what you are trying to accomplish here. Could you explain?